src/LinkDotNet.Blog.Web/Features/ShowBlogPost/ShowBlogPostPage.razor

```diff
@@ -81,8 +83,11 @@ else if (BlogPost is not null)
 				<Like BlogPost="@BlogPost" OnBlogPostLiked="@UpdateLikes"></Like>
 				<ShareBlogPost></ShareBlogPost>
 			</div>
-			<DonationSection></DonationSection>
-			@if (AppConfiguration.Value.ShowSimilarPosts)
+            @if (SupportConfiguration.Value.ShowUnderBlogPost)
+            {
+                <LinkDotNet.Blog.Web.Features.SupportMe.Components.DonationSection />   
```

Comment:
nitpick:You could import the namespace `LinkDotNet.Blog.Web.Features.SupportMe.Components` so that the tag name is only: `<DonationSection>`

Reply:
Fixed thanks